### ISSUE TITLE
Support for streaming Ogg Vorbis files

### ISSUE THREAD

**Issue by a11cf0** [BearWare/TeamTalk5#776]
Hello.
Can support for streaming Ogg Vorbis files be addet at least for Windows clients? Previously it was possible with externaal codecs but since V5.4 it's no longer working.

**Comment by Oreonan** [comment]
I'm totally agree with this, it's a real problem which cause some users to use V5.3 again today just for this feature. But, unfortunately, @bear101 said to me that it's not possible...

**Comment by kirill-jjj** [comment]
why is it not possible, you have just done the streaming in ogg opus, and Vorbis is also a free codec, if you really can't do it, what is the reason?

**Comment by zstanecic** [comment]
Yep, more arguments will be needed, othervise blacklisting old clients on a server side basis should be done...

**Comment by jeremyp3** [comment]
has this problem been forgotten? we still can't read ogg file on windows

thanks !

**Comment by kirill-jjj** [comment]
I repeat my question. @bear101, @CoBC  Why is it not possible to broadcast ogg vorbis files? licensing problems? hard code? what exactly prevents adding the most popular format after aac? you added opus with the same license.

**Comment by bear101** [comment]
@kirill-jjj You can come an babysit my kids for a week then I'll have time to look into it.

**Comment by kirill-jjj** [comment]
@bear101, sorry, I was asking about the impossibility, not why it hasn't been done yet.

**Comment by cyrmax** [comment]
Hello, @bear101.
I would be glad to babysit your kids for a some time if I had such an oportunity.
And I understand that lack of time can freeze the development.
But here we are not talking about free time to implement a feature. We are talking about the fact that you say that implementation is impossible.
I know c++ a little and cannot contribute to TeamTalk but I know at least two or three third-party libraries which could help you in decoding vorbis.

We do not force you to implement this feature ASAP. We just ask to not tell that it is impossible and put this implementation to your backlog.

**Comment by CoBC** [comment]
As this issue is still open, I think no one has sais that it is really impossible now. When I commented on this issue for the first time, I said that @bear101 told me that it is impossible, but it's my mistake, the real words it's more complicated that really impossible, so this will probably be done in the future :)

**Comment by bear101** [comment]
It will take a couple of days to implement this. Unfortunately I haven't had the time to do it.
